queries/markdown/folds.scm

```diff
@@ -0,0 +1,5 @@
+[
+  (fenced_code_block)
+  (indented_code_block)
```

Comment:
Does this make sense when header folding is not added yet because the parser doesn't parse hierarchical? Using make-range would not be very future-proof

Reply:
It's fine for me to have some basic folding, people can always use `let g:markdown_folding = 1` to fold headers, that option overrides tree-sitter folding.